content/contact/officehours.md

```diff
@@ -0,0 +1,38 @@
+{
+    "type": "office_hours",
+    "layout": "type",
+    "title": "Office Hours",
+    "linktitle": "officehours", 
+    "weight": "3"
+}
+Office hours are a chance for you to get individualized help and/or get any questions you have about Cdev answered by one of our team members.
+We have set up office hours that occur each week on the following days/times:
+
+- Mondays 12-1
+- Tuesdays 3-4
+- Wednesdays 4-5
+- Thursdays 2-3
+
+You can join these zoom meetings using this link:  [Office Hours](https://zoom/deadlink)
+{{<break 2>}}
+If these times don't fit your schedule, or you would like to schedule a 1-on-1 meeting with one of our team members you may request a meeting using one of the following calendlys:
+
+{{<break 1>}}
+
+**Daniel Sanchez**
+Founder of Cdev
+img
+calendly link
+{{<break 1>}}
+
+**Ian Magalhaes**
+Fullstack Developer

```

Comment:
Maybe add a code snippet for a card that can take a picture because I like the idea of having our headshots on their to make it more personable